client/src/cli_args.rs

```diff
@@ -32,6 +34,8 @@ pub trait EncointerArgs<'b> {
 	fn time_offset_arg(self) -> Self;
 	fn all_flag(self) -> Self;
 	fn tx_payment_cid_arg(self) -> Self;
+	fn meetup_index_arg(self) -> Self;
+	fn all_upto_arg(self) -> Self;
```

Comment:
I can live with this, but why not just --all? The cli can find out the number of assigned meetups itself

Reply:
hmm, I really would prefer this to be `--all`

Reply:
i thought that this would become a problem😅 the all flag is already taken and takes no argument this is why i did it like this. is there an easy solution to this?

Reply:
oh yes that would be possible. i misread the issue and thought that you wanted to be able to specify an upper bound